corehq/apps/userreports/pillow.py

```diff
@@ -6,6 +6,7 @@
 
 from django.conf import settings
 
+import signal
```

Comment:
This is an official python lib import. It should go just after `import hashlib` in this file https://github.com/dimagi/commcare-hq/blob/40af04629217a861696cb75f36b85bd6826df74e/.isort.cfg